docs/dictionary/control_st/if.lcdoc

```diff
@@ -4,9 +4,30 @@ Type: control structure
 
 Syntax: if <condition> then <statementList> [else <elseStatementList>]
 
+Syntax:if <condition> then
+   <statementList>
+ [else if <condition> then
+   <elseifStatementList>]
+ [else 
+   <elseStatementList>]
+end if
+
+
 Summary:
-<execute|Executes> a list of <statement|statements> if a condition is
-true. 
+Evaluates a <conditional expression|condition> to determine if it is true
+and if so <execute|executes> a subsequent <statment|statement> or <statementlist|statement list>.
+
+If the <conditional expression|condition> is false and the <if> <control structure> contains
+an <else> keyword the <statement|statement> or <statementlist|statement list> following the <else>
+<keyword> is executed.
+
+The  <if> <control structure> may contain 1 or more <else if|else if> <keyword|keywords> which have a
```

Comment:
`<else if>`

Reply:
There's an extra space here between `The` and `<if>`. Also, could you put `one` instead of `1` please?